umbraco-cms/reference/configuration/maximumuploadsizesettings.md

```diff
@@ -28,7 +28,7 @@ To configure the default 28.6MB upload limit using IIS, we have to create a web.
 `maxAllowedContentLength` is specified in bytes, so this configuration would limit requests, and therefore uploaded files, to 2 megabytes
 
 {% hint style="info" %}
-If your site is hosted on Umbraco Cloud, this is the method you should use.
+If your site is hosted on Umbraco Cloud, this is the method you should use. Umbraco Cloud uses IIS for hosting, and IIS needs the setting to be in a web.config file for this to work.
```

Comment:
```suggestion
**Are you hosting your site on Umbraco?**
Umbraco Cloud uses IIS for hosting. This means you need to add the setting in a `web.config` file for this to work on your Umbraco Cloud hosted sites.
```

Reply:
I guess it should be `**Are you hosting our site on Umbraco Cloud?**`

Reply:
😂 Oh my... Yes. that's a "typo" on my end. Thanks for keeping me on my toes 😄 I'll correct it!